sys/page_revisions.js

```diff
@@ -88,34 +237,11 @@ PRS.prototype.getTableSchema = function() {
  *  - userhidden: remove user information from response
  *
  * @param item Object the revision item
- * @return true
  * @throws HTTPError if access to the revision should be denied
  */
 PRS.prototype._checkRevReturn = function(item) {
+    mwUtil.applyAccessChecks(item);
```

Comment:
@gwicke yes. You can look at the usage example in the commented-out code section in the parsoid module.